dcm_planner:
    z_com: 0.9 #meter
    step_time: 0.75 #second
    step_size: 0.2 #meter
    n_steps: 10
    dt: 0.01 #second
    contacts: ["l_sole","r_sole"]
    l_sole:
        init_pose: [0.0,1.0,0.0,0.0,0.0,0.0,1.0]
        distal_link: ""
    r_sole:
        init_pose: [0.0,-1.0,0.0,0.0,0.0,0.0,1.0]
        distal_link: ""
